builtin/settingtypes.txt

```diff
@@ -418,7 +418,10 @@ enable_particles (Digging particles) bool true
 
 [**Filtering]
 
-#    Use mip mapping to scale textures. May slightly increase performance.
+#    Use mip mapping to scale textures. May slightly increase performance,
+#    especially when using a high resolution texture pack.
+#    Note that it currently lacks gamma correct downscaling generating the mip
```

Comment:
I don't think failings of current mipmap belongs in docs.

Reply:
Downscaling with missing gamma correction happens at many places, e.g. in firefox. I think this weakness should be documented.

Reply:
It's documented in form of an issue in this repository - it's a bug.

Reply:
Documentation is not advertising. If something lacks a feature it should be documented and properly linked to the issue until the issue was fixed.

Reply:
It doesn't help in any way to mention this in setting docs, as the user will be aware of the problem when they use it, and there's nothing they can do to avoid the problem. Many aspects of MT are imperfect but unless there is a useful reason to do so we don't write about this in docs, otherwise docs would double in size.

Reply:
2 core devs support this being removed.

Reply:
> as the user will be aware of the problem when they use it

Did anyone notice missing gamma correction in mip mapping before I did?